README.md

```diff
@@ -350,7 +350,7 @@ To update Testify to the latest version, use `go get -u github.com/stretchr/test
 Supported go versions
 ==================
 
-We currently support the most recent major Go versions from 1.19 onward.
+We currently support the most recent major Go versions from 1.17 onward.
```

Comment:
Do not change this right now. I want the CI to run first with the old versions.

Also, the versions supported is not a light policy change and other co-maintainers must be involved.

Reply:
Sure, as you can see, the CI is running for the old versions.